test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -82,28 +84,55 @@ export default class PerfCompare extends Component {
 
     handleGetPerffarmRuns = async () => {
 
-        this.setState(
-            {
-                submitStatus: 'submit',
-                progress: 20,
-                progressStatus: 'active'
-            }
-        )
+        
                 
         let resBenchmarkRuns = await fetch(`/api/getPerffarmRunCSV?${stringify(this.state.selectedRuns)}`, {
             method: 'get'
         } );
 
         const resBenchmarkRunsJson = await resBenchmarkRuns.json();
-
+       	
+       this.setState(
+            {
+                submitStatus: 'submit',
+                progress: 20,
+                progressStatus: 'active'
+            }
+        )
+       
+       if(resBenchmarkRunsJson.serverStatus === false){
+       		this.setState(
+                {
+                    inputURL: {
+                        baselineID: "",
+                        testID: "",
+                    },
+                    selectedRuns: {
+                    	baselineServerURL: "",
+                    	testServerURL: "",
+                        baselineID: "",
+                        testID: "",
+                    },
+                    submitStatus: 'none',
+                    progress: 0,
+                    progressStatus: '',
```

Comment:
I do not think we have to reset all the values.

Reply:
@sophiaxu0424 Could you please address this concern? Please ask here if you have any questions regarding this.

Reply:
Agree, we only need to reset the displayAlert part here.